pubkey_encrypt.module

```diff
@@ -74,3 +74,43 @@ function pubkey_encrypt_user_insert(\Drupal\Core\Entity\EntityInterface $entity)
   $pubkey_encrypt_manager = \Drupal::service('pubkey_encrypt.pubkey_encrypt_manager');
   $pubkey_encrypt_manager->initializeUserKeys($entity);
 }
+
+/**
+ * Implements hook_user_role_delete().
+ */
+function pubkey_encrypt_user_role_delete(\Drupal\user\RoleInterface $role) {
+  // Cater for Role deletion.
```

Comment:
A better comment here would be something like: "Delete the role keys." as it explains exactly what you're doing.